modules/op-using-remote-task-annotations-with-pipelines-as-code.adoc

```diff
@@ -0,0 +1,91 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: REFERENCE
+[id="using-remote-task-annotations-with-pipelines-as-code_{context}"]
+= Using remote task annotations with {pac} 
+
+[role="_abstract"]
+{pac} supports fetching remote tasks or pipeline using annotations in a pipeline run. If you reference a remote task in a pipeline run, or a pipeline in a `PipelineRun` or a `PipelineSpec` objects, the {pac} resolver automatically includes them. If there is any error while fetching the remote tasks or parsing them, {pac} stops processing the tasks.
+
+To include remote tasks, refer to the following examples of annotation:
+
+[discrete]
+.Reference remote tasks in {tekton-hub}
+
+* Reference a single remote task in {tekton-hub}
+
+[source,yaml]
+----
+...
+  pipelinesascode.tekton.dev/task: "git-clone" <1>
+...
+----
+<1>{pac} includes the latest version of the task from the {tekton-hub}.
+
+* Reference multiple remote tasks from {tekton-hub}
+
+[source,yaml]
+----
+...
+  pipelinesascode.tekton.dev/task: "[git-clone, golang-test, tkn]"
+...
+----
+
+* Reference multiple remote tasks from {tekton-hub} using the `-<NUMBER>` suffix
+
+[source,yaml]
+----
+...
+  pipelinesascode.tekton.dev/task: "git-clone"
+  pipelinesascode.tekton.dev/task-1: "golang-test"
+  pipelinesascode.tekton.dev/task-2: "tkn" <1>
+...
+----
+<1> By default, {pac} interprets the string as the latest task to fetch from {tekton-hub}.
+
+* Reference a specific version of a remote task from {tekton-hub}
+
+[source,yaml]
+----
+...
+  pipelinesascode.tekton.dev/task: "[git-clone:0.1]" <1>
+...
+----
+<1> Refers to the `0.1` version of the `git-clone` remote task from {tekton-hub}.
+
+[discrete]
+.Remote remote tasks using URLs
+
+[source,yaml]
+----
+...
+  pipelinesascode.tekton.dev/task: "<https://remote.url/task.yaml>" <1>
+...
+----
+<1> The public URL to the remote task.
++
+[NOTE]
+====
+* If you are using GitHub and the remote task URL uses the same host as the `Repository` CRD, {pac} uses the GitHub token and fetches the URL using the Github API.
++
+For example, if you have a repository URL similar to `https://github.com/organization/repository` and the remote HTTP URL references a GitHub blob similar to `https://github.com/organization/repository/blob/mainbranch/path/file`, {pac} fetches the task definition files from that private repository with the GitHub app token. 
```

Comment:
```suggestion
For example, if you have a repository URL similar to `https://github.com/organization/repository` and the remote HTTP URL references a GitHub blob similar to `https://github.com/organization/repository/blob/mainbranch/path/file`, {pac} fetches the task definition files from that private repository with the GitHub App token. 
```

Reply:
@Srivaralakshmi I'm sorry, could you please elaborate on your suggestion? Do you want me to use something generic? Any suggestions? Also, I believe we have referenced GitHub repositories multiple times in our Pipeline docs.

Reply:
@gtrivedi88 As per the OCP guidelines, you are not supposed to do so. PTAL: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites

Use something generic as an alternative.

Reply:
Thank you @Srivaralakshmi 
As discussed, I'll be adding the `<>` brackets.
Comment addressed!